### ISSUE TITLE
New DNS Forwarding Ruleset Rule Examples are incomplete

### ISSUE THREAD

**Issue by mundayn** [Azure/azure-powershell#24704]
### Type of issue

Missing information

### Feedback

Hi all,

This documentation is unclear and hard to follow.

Please see this page that provides a very clear example on how to use "New-AzDnsForwardingRulesetForwardingRule":
https://learn.microsoft.com/en-us/azure/dns/dns-private-resolver-get-started-powershell#create-forwarding-rules

The page in question does not show you how to add a "DomainName" value for the rule, which is the a core part of the rule.

The "$targetIPConfig" also does not make any sense to me to why it is needed. I have a feeling this is an error, and it should be "New-AzDnsResolverTargetDnsServerObject", and not "New-AzDnsResolverIPConfigurationObject".

### Page URL

https://learn.microsoft.com/en-us/powershell/module/az.dnsresolver/new-azdnsforwardingrulesetforwardingrule?view=azps-11.5.0

### Content source URL

https://github.com/Azure/azure-powershell/blob/main/src/DnsResolver/DnsResolver/help/New-AzDnsForwardingRulesetForwardingRule.md

### Author

@mikefrobbins

### Document Id

66bb54a2-a30b-8869-f36b-6949baef5627

**Comment by isra-fel** [comment]
Thanks for reporting. Let me loop in DNS team to follow up.

**Comment by argagnej** [comment]
Checked the latest doc, https://learn.microsoft.com/en-us/azure/dns/dns-private-resolver-get-started-powershell#create-forwarding-rules, and it does have the information on how to add a domain name.

<img width="1147" height="726" alt="Image" src="https://github.com/user-attachments/assets/6d762cf0-064b-4c6d-a0aa-ce11f37e6851" />

**Comment by mundayn** [comment]
> Checked the latest doc, https://learn.microsoft.com/en-us/azure/dns/dns-private-resolver-get-started-powershell#create-forwarding-rules, and it does have the information on how to add a domain name.
> 
> <img alt="Image" width="1147" height="726" src="https://private-user-images.githubusercontent.com/171605165/532100192-6d762cf0-064b-4c6d-a0aa-ce11f37e6851.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njc2NTQyNTUsIm5iZiI6MTc2NzY1Mzk1NSwicGF0aCI6Ii8xNzE2MDUxNjUvNTMyMTAwMTkyLTZkNzYyY2YwLTA2NGItNGM2ZC1hMGFhLWNlMTFmMzdlNjg1MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEwNVQyMjU5MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YzdiOTAyOTI0Yjc5ZTdmZGMyNjFkZWE4MjU3ZDBmMDBjYTU4MjI4MTdmOWNhYmMwN2VlMDNlM2Y0ODY5MDgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.nyiCpCLJUrSt-EEqdE-KTlGJ67Py-8OgjErTqzCgrFc">

Hi

The quickstart page does advise, but the official documentation page for the cmdlet does not.

Page in questions:
https://learn.microsoft.com/en-us/powershell/module/az.dnsresolver/new-azdnsforwardingrulesetforwardingrule?view=azps-15.1.0&viewFallbackFrom=azps-11.5.0

**Comment by argagnej** [comment]
Thanks @mundayn for bringing this to our attention. We will fix the documentation.
